packages/discovery/src/DiscoveryItems.php

```diff
@@ -17,6 +17,20 @@ public function __construct(
         private array $items = [],
     ) {}
 
+    public function addForLocation(DiscoveryLocation $location, array $values): self
```

Comment:
Discovery items were already grouped by location, but those weren't exposed to the outside.